### ISSUE TITLE
KSQL throws "Invalid Expression" error on wildcard column select if stream/table is aliased

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#1446]
```
ksql> SELECT S.* FROM SYSLOG S LIMIT 1;
Invalid Expression SYSLOG.ROWTIME.
ksql>
ksql> SELECT * FROM SYSLOG S LIMIT 1;
Invalid Expression SYSLOG.ROWTIME.
ksql>
```

**Comment by big-andy-coates** [comment]
This would be a perfect issue to include a test defined in Json, (as discussed offline)

**Comment by rmoff** [comment]
Related? #2176

**Comment by vcrfxia** [comment]
This is fixed on the current `master` but not `5.3.x`, so it'll be picked up in the next release. Would be good to add a query validation test for it (as @big-andy-coates mentioned above). Will let @stevenpyzhang tackle that and then close out this issue.
